components/datetime/src/mock/mod.rs

```diff
@@ -25,6 +25,17 @@ pub mod zoned_datetime;
 /// let date: DateTime<Gregorian> = parse_gregorian_from_str("2020-10-14T13:21:00")
 ///     .expect("Failed to parse a datetime.");
 /// ```
+///
+/// Optionally, fractional seconds can be specified: `YYYY-MM-DDThh:mm:ss.SSSS`.
+///
+/// ```
+/// use icu::datetime::mock::parse_gregorian_from_str;
+/// use icu_calendar::{DateTime, Gregorian};
+///
+/// let date: DateTime<Gregorian> = parse_gregorian_from_str("2020-10-14T13:21:00.101")
+///     .expect("Failed to parse a datetime.");
+/// assert_eq!(u32::from(date.time.fraction), 101);
```

Comment:
Question: if fractional seconds are in nanoseconds, then shouldn't `00.101` turn into 101e6?

Reply:
Yes, you're right, it should be 101e6.